packages/react-reconciler/src/ReactFiber.js

```diff
@@ -366,42 +367,9 @@ export function createFiberFromElement(
         // mode compatible.
         mode |= StrictMode;
         break;
-      default: {
```

Comment:
I had to extract this because Flow couldn't follow that `fiberTag` is always assigned in practice.